mogc_partnerships/views.py

```diff
@@ -290,8 +290,6 @@ def unenroll(self, cohort_member):
         if not eligible_enrollment_records:
             return
 
-        eligible_enrollment_records.update(is_active=False)
```

Comment:
Oh - I expected this result to be static here, this must have been changing the value of `eligible_enrollment_records` and after the update there weren't eligible records anymore.